File: rumahsehat_flutter/lib/DTO/TagihanDTO.dart
// Yang belum: tanggaldibuat tanggal dibayar, dan appoinment

class TagihanDTO {
  final String nomorTagihan;
  final String tanggalTerbuat;
  final String status;
  final String idPasien;
  final String tanggalBayar;
  final int jumlahTagihan;

  TagihanDTO(this.nomorTagihan, this.tanggalTerbuat, this.status, this.idPasien,
      this.jumlahTagihan, this.tanggalBayar);
}

File: rumahsehat/src/main/java/apap/tugas/akhir/rumahsehat/model/DTO/TagihanDTO.java
package apap.tugas.akhir.rumahsehat.model.DTO;

import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
public class TagihanDTO {
    private String nomorTagihan;
    private String tanggalTerbuat;
    private String status;
    private String idPasien;
    private int jumlahTagihan;
    private String tanggalBayar;
}
